### PR TITLE
STY: flake8 -- sw instruments

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Removes the flake8 exceptions for:
- instruments/sw_dst
- instruments/sw_f107
- instruments/sw_kp

Scrubs the code for flake8 compliance.

## Type of change

- Style fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Via pytest --flake8

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: pysat/instruments/sw_dst.py [hunk 1]
```diff
@@ -215,7 +216,7 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
         local_fname = fname
         saved_fname = os.path.join(data_path, local_fname)
         try:
-            logger.info('Downloading file for '+date.strftime('%D'))
+            logger.info('Downloading file for ' + date.strftime('%D'))
```
Comment (aburrell) [inline pysat/instruments/sw_dst.py:219]:
```suggestion
            logger.info(''.join(['Downloading file for ', date.strftime('%D')]))
```

File: pysat/instruments/sw_f107.py [hunk 2]
```diff
@@ -525,7 +530,8 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
                     break
 
             if not downloaded:
-                logger.info('File not available for {:}'.format(date.strftime('%x')))
+                logger.info(' '.join(('File not available for',
+                                      '{:}'.format(date.strftime('%x')))))
```
Comment (aburrell) [inline pysat/instruments/sw_f107.py:534]:
```suggestion
                                      date.strftime('%x'))))
```

File: pysat/instruments/sw_f107.py [hunk 3]
```diff
@@ -613,8 +621,8 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
         data = pds.DataFrame(f107, index=f107_times, columns=['f107'])
         data['ap'] = ap
         # write out as a file
-        data.to_csv(os.path.join(data_path, 'f107_45day_' +
-                                 date.strftime('%Y-%m-%d') + '.txt'),
+        data.to_csv(os.path.join(data_path, 'f107_45day_'
+                                 + date.strftime('%Y-%m-%d') + '.txt'),
```
Comment (aburrell) [inline pysat/instruments/sw_f107.py:625]:
```suggestion
        data_file = 'f107_45day_{:s}.txt'.format(date.strftime('%Y-%m-%d'))
        data.to_csv(os.path.join(data_path, data_file),
```

File: pysat/instruments/sw_kp.py [hunk 4]
```diff
@@ -293,15 +293,17 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
 
         for date in date_array:
             fname = 'kp{year:02d}{month:02d}.tab'
-            fname = fname.format(year=(date.year - date.year//100*100),
+            fname = fname.format(year=(date.year - date.year // 100 * 100),
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:296]:
```suggestion
            fname = fname.format(year=(date.year - (date.year // 100) * 100),
```

Comment (aburrell) [reply pysat/instruments/sw_kp.py:296]: For clarity.

File: pysat/instruments/sw_kp.py [hunk 5]
```diff
@@ -356,17 +359,18 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
         # put data into nicer DataFrame
         data = pds.DataFrame(day, index=times, columns=['Kp'])
         # write out as a file
-        data.to_csv(os.path.join(data_path, 'kp_forecast_' +
-                                 date.strftime('%Y-%m-%d') + '.txt'),
+        data.to_csv(os.path.join(data_path, ''.join(('kp_forecast_',
+                                                     date.strftime('%Y-%m-%d'),
+                                                     '.txt'))),
                     header=True)
```
Comment (aburrell) [inline pysat/instruments/sw_kp.py:365]:
```suggestion
        data_file = 'kp_forecast_{:s}.txt'.format(date.strftime('%Y-%m-%d'))
        data.to_csv(os.path.join(data_path, data_file), header=True)
```

**Comment by jklenzing** [comment]
Thanks!

File: pysat/instruments/sw_f107.py [hunk 6]
```diff
@@ -427,8 +430,9 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
         idx, = np.where(data['f107'] == -99999.0)
         data.iloc[idx, :] = np.nan
         # create file
-        data.to_csv(os.path.join(data_path, 'f107_1947_to_' +
-                                 now.strftime('%Y-%m-%d') + '.txt'),
+        data.to_csv(os.path.join(data_path, ''.join(('f107_1947_to_',
+                                                     now.strftime('%Y-%m-%d'),
+                                                     '.txt'))),
```
Comment (aburrell) [inline pysat/instruments/sw_f107.py:435]: One more file simplification.

Comment (jklenzing) [reply pysat/instruments/sw_f107.py]: found some more in this file too

**Comment by jklenzing** [comment]
ADDENDUM: ran test_instruments.py locally to ensure ftp instruments are working properly.

**Pull request merged**